GetDKAN/dkan2
# Move from profile to modules

dkan-tools PR https://github.com/GetDKAN/dkan-tools/pull/179

- removes lunr
- moves dkan config to modules
- remove reference to profile in SchemaRetriever.php
- Temp. use 359-composer-merge branch of dkan-tools in circleci/config.yml
- update dredd.yml
- update dkan_metastore_form build files (this is why the files changed count is so high)
- code climate issues ticketed: https://github.com/GetDKAN/dkan2/issues/377

## QA steps
- Checkout dkan-tools/pull/179
- Run `dktl get 8.8.5`
- `dktl make --frontend --branch=359-deprofiling`
- `dktl install --frontend`
- `dktl drush dkan-dummy-content:create`
- `dktl drush dkan-search:rebuild-tracker`
- `dktl drush sapi-i`
- You should see the dkan code in /docroot/modules/contrib/dkan2
- Confirm the /admin/content page is the custom dkan version
- /admin/content/datasets should function as before
- harvest/datstore/api all working as expected
